lambdas/process-journey-step/src/main/resources/statemachine/build/ipv-core-main-journey.yaml

```diff
@@ -58,6 +58,17 @@ MITIGATION_JOURNEY:
 INITIAL_IPV_JOURNEY:
   name: INIT
   parent: null
+  events:
+    next:
+      type: basic
+      name: next
+      targetState: CHECK_EXISTING_IDENTITY
```

Comment:
I think this event is being applied to the wrong state. Currently this means we are checking the for an existing identity as the very first thing in IPV Core. But from the figma journey diagram, I think we first need to display the IPV_IDENTITY_START_PAGE then do the check after that

Reply:
Yeah, sorry, I think my description was wrong in the ticket was wrong.

Reply:
I've updated the ticket with the correct description (I think).

Reply:
Corrected this!